libraries/abstractions/common_io/include/iot_uart.h

```diff
@@ -47,34 +47,35 @@
 /**
  * @brief Error codes returned by every function module in UART HAL.
  */
-#define IOT_UART_SUCCESS                   ( 0 )
-#define IOT_UART_INVALID_VALUE             ( 1 )
-#define IOT_UART_WRITE_FAILED              ( 2 )
-#define IOT_UART_READ_FAILED               ( 3 )
-#define IOT_UART_BUSY                      ( 4 )
-#define IOT_UART_NOTHING_TO_CANCEL         ( 5 )
-#define IOT_UART_FUNCTION_NOT_SUPPORTED    ( 6 )
+#define IOT_UART_SUCCESS                   ( 0 ) /*!< UART operation completed successfully. */
+#define IOT_UART_INVALID_VALUE             ( 1 ) /*!< At least one parameter is invalid. */
+#define IOT_UART_WRITE_FAILED              ( 2 ) /*!< UART driver returns error when performing write operation. */
+#define IOT_UART_READ_FAILED               ( 3 ) /*!< UART driver returns error when performing read operation. */
+#define IOT_UART_BUSY                      ( 4 ) /*!< UART bus is busy at current time. */
+#define IOT_UART_NOTHING_TO_CANCEL         ( 5 ) /*!< No ongoing receiving or transmitting when cancel operation is performed. */
```

Comment:
nit: operation instead of receiving or transmitting

Reply:
ok